WordPress/Classes/ViewRelated/Comments/EditCommentTableViewController.swift

```diff
@@ -19,8 +19,17 @@ class EditCommentTableViewController: UITableViewController {
     // So save and use one instance of the cell.
     private let commentContentCell = EditCommentMultiLineCell.loadFromNib()
 
+    // A closure executed when the view is dismissed.
+    // Returns the Comment object and a Bool indicating if the Comment has been changed.
+    @objc var completion: ((Comment, Bool) -> Void)?
+
     // MARK: - Init
 
+    convenience init(comment: Comment, completion: ((Comment, Bool) -> Void)? = nil) {
```

Comment:
I kinda jumped the gun on this. It's not used in this PR, but it is in the [next](https://github.com/wordpress-mobile/WordPress-iOS/pull/17099).